src/main/java/appeng/client/render/items/ToolColorApplicatorRender.java

```diff
@@ -58,9 +58,10 @@ public void renderItem( ItemRenderType type, ItemStack item, Object... data )
 		float f7 = par2Icon.getMaxV();
 		float f12 = 0.0625F;
 
-		Tessellator tessellator = Tessellator.instance;
+		final Tessellator tessellator = Tessellator.instance;
+
 		GL11.glPushMatrix();
-		GL11.glPushAttrib( GL11.GL_ALL_ATTRIB_BITS );
+		// GL11.glPushAttrib( GL11.GL_ALL_ATTRIB_BITS );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)